include/nlohmann/detail/output/binary_writer.hpp

```diff
@@ -1625,7 +1625,7 @@ class binary_writer
     }
 
   private:
-    /// whether we can assume little endianess
+    /// whether we can assume little endianness
     const bool is_little_endian = little_endianess();
```

Comment:
Please rename this function and its calls to `little_endianness`.

Reply:
Done!